### PR TITLE
DM lower/uppercase fixes, abnormality descriptions 

### PR DESCRIPTION
**Pull request by badlypayedgoon**
## About The Pull Request
Streamlines all of the Abnormality names for lower/uppercase usage. A few name fixes and/or description changes. **No** actual code has been changed.

## Why It's Good For The Game
Because grammar matters.

## Changelog
:cl:
tweak: tweaked a few things
fix: fixed a few things
spellcheck: fixed a few typos

/:cl:

### CODE REVIEW

File: code/datums/ai/sanity/sanityloss_behaviors.dm [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-				"It will end quickly, so relax. I�ll free you from this prison we call flesh."
+				"It will end quickly, so relax. I'll free you from this prison we call flesh."
 				)
 
 /datum/ai_behavior/say_line/insanity_suicide
 	lines = list(
-				"It's all my fault. It�s my responsibility...",
-				"I can hear someone. It�s the sound of back home. I just can�t stop hearing it�",
+				"It's all my fault. It's my responsibility...",
+				"I can hear someone. It's the sound of back home. I just can't stop hearing it�",
```
Comment (EgorDinamit) [inline code/datums/ai/sanity/sanityloss_behaviors.dm:13]: Undefined symbol..?

Comment (badlypayedgoon) [reply code/datums/ai/sanity/sanityloss_behaviors.dm:13]: The " ’ " were being shown as questionmarks in the code.  I will fix the ? real quick.

File: code/modules/mob/living/simple_animal/abnormality/aleph/blue_star.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /mob/living/simple_animal/hostile/abnormality/bluestar
-	name = "Blue star"
-	desc = "Floating heart-shaped object. It's alive, and soon you will become one with it."
+	name = "Blue Star"
+	desc = "Let us meet again as stars."
```
[the remark this answers is not in the document]
Comment (badlypayedgoon) [reply code/modules/mob/living/simple_animal/abnormality/aleph/blue_star.dm:3]: Fair enough. I felt like the second sentence went against the overall flair of Lobotomy Corporation (as in being too direct and blunt).

**Comment by EgorDinamit** [comment]
Conflicts.
Also item names shouldn't be uppercase.

**Comment by EgorDinamit** [comment]
Conflicts :skull:

**Pull request closed without merging**